daemon/station_reporter.rs

```diff
@@ -28,15 +28,23 @@ impl StationReporter {
             .unwrap()
             .total_jobs_completed;
 
-        Self {
+        let reporter = Self {
             tracker: RefCell::new(JobCompletionTracker::new(
                 initial_job_count,
                 job_report_delay,
             )),
             module_name,
             log_target,
             state_file,
+        };
+
+        // Report the initial job count to prevent Station Desktop from showing incorrect job count
+        // until a Zinnia module completes the first job
+        if initial_job_count > 0 {
+            reporter.print_jobs_completed(initial_job_count);
         }
```

Comment:
```suggestion
        reporter.print_jobs_completed(initial_job_count);
```

This would be even better for Desktop, because it will know that that module's job count isn't loading any more, even when it's zero